var/spack/repos/builtin/packages/sepp/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Sepp(Package):
```

Comment:
Have you tried using the normal `PythonPackage` base class?

Reply:
I did. It does not install nicely with pip.